### ISSUE TITLE
Annotations displaced when zooming out in browser

### ISSUE THREAD

**Issue by kimpham54** [CtrHellenicStudies/OpenVideoAnnotation#9]
Description
When zooming out on a video, annotations are displaced and positioned above the video.

Steps to Reproduce
1. Create a number of annotations on a video
2. Zoom out in your Chrome browser

Expected Behaviour
Annotations resize according to the video player

Actual Behaviour
Annotations appear above the video, does not resize correctly and is positioned above the video player

Chrome browser, 75% zoom
<img width="1084" alt="screen shot 2017-04-21 at 12 13 38" src="https://cloud.githubusercontent.com/assets/2915770/25286505/fe36d956-268b-11e7-9b85-7cb6d2c63a95.png">

Notes
This only happens in Chrome (Version 57.0.2987.133 (64-bit)), this is not happening in Firefox (ver 52.0.2 (64-bit))

**Comment by danielcebrian** [comment]
It is weird, it is not happen in my project: https://gteavirtual.org/ova/index.php?r=site/page&view=explore. Try to get this version of OVA and see if it happens again.

https://gteavirtual.org/ova/js/ova.js

Best,
Daniel

**Comment by kimpham54** [comment]
@danielcebrian i can reproduce the issue in your demo on my chrome
<img width="1216" alt="screen shot 2017-04-23 at 07 37 21" src="https://cloud.githubusercontent.com/assets/2915770/25313230/cb4e4fce-27f8-11e7-94f3-8d43ea8652a5.png">

but not in firefox
<img width="892" alt="screen shot 2017-04-23 at 07 38 04" src="https://cloud.githubusercontent.com/assets/2915770/25313232/d39ec924-27f8-11e7-89d9-0f813345776a.png">

another thing i've noticed:
1. start zoomed out in your browser (try at 50%)
2. try to view annotations
<img width="672" alt="screen shot 2017-04-23 at 07 40 50" src="https://cloud.githubusercontent.com/assets/2915770/25313238/e98e2306-27f8-11e7-9ef5-f99cc21ded42.png">
3. zoom back to 100% . you will only see that the portion of annotations that you saw when you were zoomed out at 50%
<img width="852" alt="screen shot 2017-04-23 at 07 42 33" src="https://cloud.githubusercontent.com/assets/2915770/25313240/f7cd44c4-27f8-11e7-82ab-06e30ea697be.png">

by the way we love this project. thank you!

**Comment by danielcebrian** [comment]
you are right. I could reproduce right now. Thank you for letting me know it. If it is not a emergency I will fix it in some days/weeks, because I have too much work right now. I will let you know.

Best,
Daniel

**Comment by kimpham54** [comment]
it's not a pressing need but the fix will definitely be useful. thank you!
